src/emails/templates/breachAlert/BreachAlertEmail.tsx

```diff
@@ -62,7 +62,10 @@ export const BreachAlertEmail = (props: BreachAlertEmailProps) => {
   }
 
   const premiumSubscriptionUrlObject = new URL(
-    getPremiumSubscriptionUrl({ type: "yearly" }),
+    getPremiumSubscriptionUrl({
+      type: "yearly",
+      enabledFeatureFlags: [],
```

Comment:
Thinking about handling the updated URLs to support emails that have been sent out best. Most likely we’ll need to add a redirect to properly support toggling the flag between SubPlat 2 and SubPlat 3.

Reply:
Oh yeah that will be problematic - but the existing link is not to the Monitor site, right? So could we have FxA do the redirect? Or at least discuss the problem with them, see if they have a solution?

Reply:
(I assume this is the reason you're not actually passing the feature flags to the email? It might be a good idea to do it anyway, or at least call it out in a comment, because future contributors could see the `enabledFeatureFlags` parameter and expect that they can use it.